ovs/extensions/services/service.py

```diff
@@ -60,3 +80,54 @@ def __getattr__(cls, item):
             return getattr(ServiceManager.ImplementationClass, item)
 
     __metaclass__ = MetaClass
+
+    @staticmethod
+    def reload():
+        ServiceManager.ImplementationClass = None
+
+    @staticmethod
+    def has_fleet_client():
+        return HAS_FLEET_CLIENT
+
+    @staticmethod
+    def setup_fleet():
+        if HAS_FLEET_CLIENT is False:
+            logger.info('Cannot use fleet because the client is not installed')
+            return False
+        if ServiceManager.has_fleet():
+            if ServiceManager._is_fleet_running_and_usable():
+                logger.info('Fleet service is running')
+                ServiceManager.reload()
+                return True
+            else:
```

Comment:
The else is when the fleet service is detected but it's not yet running and usable, so we try to start it.